packs/alertmanager/variables.hcl

```diff
@@ -0,0 +1,135 @@
+  description = "The name to use as the job name which overrides using the pack name."
+  type        = string
+  // If "", the pack name will be used
+  default = ""
+}
+
+variable "count" {
+  description = "How many instances to start."
+  type        = string
+  default = "1"
+}
+
+variable "constraints" {
+  description = "Constraints to apply to the entire job."
+  type        = list(object({
+    attribute = string
+    operator  = string
+    value     = string
+  }))
+  default = [
+    {
+      attribute = "$${attr.kernel.name}",
+      value     = "linux",
+      operator  = "",
+    },
+  ]
+}
+
+variable "container_args" {
+  description = "Arguments passed to the alertmanager docker container"
+  type        = list(string)
+  default = [
+    "--config.file=/etc/alertmanager/config/alertmanager.yml",
+  ]
+}
+
+variable "datacenters" {
+  description = "A list of datacenters in the region which are eligible for task placement."
+  type        = list(string)
+  default     = ["dc1"]
+}
+
+variable "region" {
+  description = "The region where the job should be placed."
+  type        = string
+  default     = "global"
+}
+
+variable "version_tag" {
+  description = "The docker image version. For options, see https://hub.docker.com/r/prom/alertmanager"
+  type        = string
+  default     = "latest"
+}
+
+variable "http_port" {
+  description = "The Nomad client port that routes to the alertmanager."
+  type        = number
+  default     = 9093
+}
+
+variable "cluster_port" {
+  description = "The Nomad client port that routes to the alertmanager."
+  type        = number
+  default     = 9094
+}
+
+variable "resources" {
+  description = "The resource to assign to the alertmanager service task."
+  type = object({
+    cpu    = number
+    memory = number
+  })
+  default = {
+    cpu    = 200,
+    memory = 256
+  }
+}
+
+variable "register_consul_service" {
+  description = "If you want to register a consul service for the job"
+  type        = bool
+  default     = true
+}
+
+variable "register_consul_connect_enabled" {
+  description = "If you want to run the consul service with connect enabled. This will only work with register_consul_service = true"
+  type        = bool
+  default     = true
+}
+
+variable "consul_service_name" {
+  description = "The consul service name for the hello-world application"
+  type        = string
+  default     = "alertmanager"
+}
+
+variable "consul_service_tags" {
+  description = "The consul service name for the hello-world application"
+  type        = list(string)
+  // defaults to integrate with Fabio or Traefik
+  // This routes at the root path "/", to route to this service from
+  // another path, change "urlprefix-/" to "urlprefix-/<PATH>" and
+  // "traefik.http.routers.http.rule=Path(`/`)" to
+  // "traefik.http.routers.http.rule=Path(`/<PATH>`)"
+  default = [
+    "urlprefix-/",
+    "traefik.enable=true",
+    "traefik.http.routers.http.rule=Path(`/`)",
+  ]
```

Comment:
We don't default these tags in other packs, so I wonder if we want to do this, or default to empty with the documentation detailing how to add these? Additionally, this would be a catchall for the router so wouldn't work where other jobs are also using the router.

Reply:
actually you do, its in the hello-world and the simple-service. it now also is in the tempo pack. But happy to make it an empty array

Reply:
moved into the service object, default is now an empty array